heat-stack/tests/e2e/energy-analysis.test.ts

```diff
@@ -49,4 +49,50 @@ test('Logged out user can upload CSV, toggle table row checkbox, expecting analy
 	
 	// expect not equal
 	expect(tableHeaderContentAfterClick).not.toEqual(tableHeaderContentBeforeClick);
-})
\ No newline at end of file
+})
+
+test('Custom name persists after form submission', async ({ page }) => {
+	// Visit the root
+	await page.goto('/')
+
+	// click the "demo data" link
+	await page.getByText('Get Started (with Demo Data)').click()
```

Comment:
Ethan would like to identify the button by a way other than the text. Get by test id might be preferable in a separate PR.